tests/boot/snapper_rollback.pm

```diff
@@ -19,7 +19,13 @@ sub run() {
     my ($self) = @_;
 
     # assert the we are on a ro snapshot.
-    assert_screen 'linux-login', 200;
+    # assert_screen 'linux-login', 200;
+    # workaround known issue: bsc#980337
+    if (!check_screen('linux-login', 200)) {
+        record_soft_failure 'bsc#980337';
+        send_key "ctrl-alt-f1";
+        assert_screen 'tty1-selected';
+    }
```

Comment:
what I think can be improved:

* delete disabled code
* do not waste 200s waiting by using a multiple tag assert_screen + match_has_tag
* use "select_console" instead of "ctrl-alt-f1"
* do not repeat the bugref in the comment stating the obvious. "record_soft_failure 'bsc+980337';" is certainly stating enough